README.md

```diff
@@ -40,6 +40,7 @@ OKTA_URL_ISSUER=
 
 - run: `npm install` to download all dependencies.
 - run: `cp .env.sample .env` and update the enviornment variables to match your local setup.
+- run: `docker-compose up` * Will take a little while * (after installing Docker.)
```

Comment:
The instructions were a bit confusing without this line.

I would also note this line is meant to start the PostgreSQL DB.

Reply:
Okay. I'll do it now.